protocol/v2/ssv/runner/committee.go

```diff
@@ -414,95 +415,99 @@ func (cr *CommitteeRunner) ProcessPostConsensus(logger *zap.Logger, signedMsg *s
 			}
 		}
 	}
-
+	cr.metrics.EndPostConsensus()
+	durationFields = append(durationFields, fields.PostConsensusTime(cr.metrics.GetPostConsensusTime()))
```

Comment:
this looks better [here](https://github.com/ssvlabs/ssv/blob/cbe62540c5d2b00a97095e11b4975b196f07352d/protocol/v2/ssv/runner/attester.go#L178)
you don't have to use `durationFields`